azure-pipelines.yml

```diff
@@ -207,12 +207,12 @@ jobs:
   - template: ci/checkout-template.yml
   - template: ci/install-package-dependencies.yml
   - task: DownloadSecureFile@1
-    condition: eq(variables['Build.SourceBranchName'], 'master')
+    condition: eq(variables['Build.SourceBranchName'], 'earlgrey_es')
```

Comment:
Should we avoid letting earlgrey_es write to the bazel cache used by master?

Reply:
Done